### ISSUE TITLE
Unable to unfavorite default avatar

### ISSUE THREAD

**Issue by shiftyscales** [Neos-Metaverse/NeosPublic#2639]
## Describe the bug
Pressing the favorite buton on a favorited avatar will properly remove the purple indicator marking the avatar as a favorite, but respawning, creating a new world, or even restarting Neos will still retain the previously favorited avatar as the default avatar instead of the head and hands avatars from the SpawnObjects.

## Relevant issues
N/A

## To Reproduce
- Have a favorited avatar you spawn into by default
- Unfavorite it
- Respawn/try to use the default head and hands avatar

## Expected behavior
After unfavoriting the avatar, the next respawn should use the default SpawnObjects avatars.

## Log Files
Logs appeared to contain no relevant information/exceptions.

## Screenshots / Video
N/A

## Bug information (please complete the following information):
 - How often does it happen: Always
 - Does the bug persist after restarting Neos? Yes
 - Neos Version: 2021.7.11.1002
 - Neos Platform: Windows
 - Link to reproduction World/Item: N/A
 - Regression: Yes, I believe it was last known working prior to `2021.5.22.1024`:
 ```
 - Improved security if favorite avatars using scoped temporary access keys
-- Favorited avatar records are no longer marked as public to severely mitigate possibility of malicious access
-- In a few days we'll automatically unmark all avatar records as public unless they're in a public folder
-- Note that when unregistered user hosts, they won't be able to load private favorited avatars for you, you'll have to equip them yourself
```

## Additional context
This was observed while trying to unfavorite my avatar so I could get a look at the new hand models outside of my local home.

## Reporters:
Shifty# 7944

**Comment by shadowpanther** [comment]
As a workaround, I was able to reset the favourite avatar by saving a new copy of the avatar, favouriting it and deleting it.

**Comment by epicEaston197** [comment]
I noticed this only happens if you are the host if you're not then you can unfavorite and the expected behavior happens

**Comment by shiftyscales** [comment]
> 
> 
> I noticed this only happens if you are the host if you're not then you can unfavorite and the expected behavior happens
> 
> so I recommend you do the same thing but not be the host

Oh? That's bizarre. Thanks for letting us know that, Easton. That'll likely help our developers in isolating the issue if that's the case.

**Comment by Frooxius** [comment]
Fixed in 2021.7.21.127! It was because "null" value wouldn't work for Uri cloud variables.

I don't see how it would affect only when you're host, the system doesn't have much to do with that. My guess is that it was just coincidence.

Regardless should be fixed, now, let me know if you run into any more issues!
